### ISSUE TITLE
Build SDK doesn't work

### ISSUE THREAD

**Issue by ballen127** [frida/frida#1506]
Hello,

I tried to build frida SDK via this command.
`make -f Makefile.sdk.mk FRIDA_HOST=linux-x86_64`
But it's not working.
Please help me.

**Comment by oleavr** [comment]
This is temporarily broken as I'm working on upgrading the dependencies [here](https://github.com/frida/frida/tree/upgrade-dependencies). The current `Makefile.sdk.mk` tries to build the default branch of each repo, which means it's not possible to build SDKs for old releases. (Without manually patching `Makefile.sdk.mk` to make it `git checkout` in each repo.)
I'm finally fixing this long-standing issue in the `upgrade-dependencies` branch. Should hopefully get things back to a working state either today or tomorrow.

**Comment by ballen127** [comment]
Hi @oleavr  is it fixed? It seems that the error is gone now.

**Comment by oleavr** [comment]
> Hi @oleavr  is it fixed? It seems that the error is gone now.

Yes in theory. Use the `upgrade-dependencies` branch, and build the toolchain for the build machine first: `make -f Makefile.toolchain.mk`.

**Comment by ballen127** [comment]
Okay, thank you.
I will try now.

**Comment by ballen127** [comment]
@oleavr based what version is the `upgrade-dependencies` branch?
and is it possible to build version 12.x without error?

**Comment by ballen127** [comment]
@oleavr I tried `make -f Makefile.toolchain.mk` but i got this error.
`make: *** No rule to make target 'build/ft-tmp-linux-x86_64/.package-stamp', needed by 'build/toolchain-linux-x86_64.tar.bz2'.  Stop.`

**Comment by ballen127** [comment]
Hello @oleavr I am getting an error during toolchain build with `upgrade-dependecies` branch.

**Comment by oleavr** [comment]
@ballen127 Yeah I hadn't gotten to Linux yet. Should be in better shape now. (But still building so might discover more issues.)

**Comment by ballen127** [comment]
@oleavr I can get SDK and toolchain build done without any error. But I got error on frida-gum build.

**Comment by ballen127** [comment]
@oleavr I could compile frida-server and 64bit linux devkit.
And I ran the compiled frida-server in Ubuntu server and I tried to attach process remotely.
But i am getting this error.
`Failed to attach: remote-exec PTRACE_CONT wait failed: 0`
Can you please help me why i am getting this error?

**Comment by ballen127** [comment]
@oleavr I could get SDK and toolchain build done without any error in Ubuntu 18.04.
But I got this error during `frida-core` build - `make build/frida-linux-x86_64/lib/pkgconfig/frida-core-1.0.pc`
```
/bin/sh: 1: .: Can't open build/frida-meson-env-linux-x86.rc
Makefile.linux.mk:113: recipe for target 'build/frida-linux-x86/lib/pkgconfig/frida-gum-1.0.pc' failed
make[1]: *** [build/frida-linux-x86/lib/pkgconfig/frida-gum-1.0.pc] Error 2
```
